include/conversation.php

```diff
@@ -834,16 +856,17 @@ function conversation_add_children(array $parents, $block_authors, $order, $uid)
  * @param array $condition
  * @param boolean $block_authors
  * @param array $params
+ * @param integer $causer
```

Comment:
Please add a description for this parameter (and all the others while you're at it 😇)